### PR TITLE
Submit hw2

### PR DESCRIPTION
**Pull request by iblislin**
A good PR should include the following parts:

- [x] A clear title
- [x] A meaningful message for PR, as well as its commits
- [x] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [x] excluding any irrelevant files, such as binaries, text files, or dot files
- [x] passing tests/CI

### CODE REVIEW

File: hw2/iblis17/q2/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+*.py[oc]
```
Comment (tigercosmos) [inline hw2/iblis17/q2/.gitignore:1]: don't upload garbage.

Comment (iblislin) [reply hw2/iblis17/q2/.gitignore:1]: well, I think the committing the gitignore file is acceptable in a git repo.

Comment (tigercosmos) [reply hw2/iblis17/q2/.gitignore:1]: > excluding any irrelevant files, such as binaries, text files, or dot files

Comment (tigercosmos) [reply hw2/iblis17/q2/.gitignore:1]: yes. gitignore is important.
However, you should only upload code for assigments.

Comment (iblislin) [reply hw2/iblis17/q2/.gitignore:1]: :ok_hand:

**Pull request merged**